gobblin-service/src/main/java/org/apache/gobblin/service/modules/core/GobblinServiceManager.java

```diff
@@ -117,7 +117,7 @@ public class GobblinServiceManager implements ApplicationLauncher, StandardMetri
   public static final String SERVICE_EVENT_BUS_NAME = "GobblinServiceManagerEventBus";
 
   private static final Logger LOGGER = LoggerFactory.getLogger(GobblinServiceManager.class);
-  private static volatile GobblinServiceGuiceModule GOBBLIN_SERVICE_GUICE_MODULE;
+  @Setter private static volatile GobblinServiceGuiceModule GOBBLIN_SERVICE_GUICE_MODULE;
```

Comment:
not seeing where this is used... did I miss something?

Reply:
it will be used in li-gobblin.
```
* If {@link GobblinServiceManager} is created using guice, user should set {@link GobblinServiceManager#GOBBLIN_SERVICE_GUICE_MODULE}
   * for this method to work.
```
is the for that li-gobblin code

Reply:
gotcha.  probably deserves a comment like:
```
`@Setter` for use by downstream deployment code (beyond this repo)
```